survey-sdk/src/main/java/co/nimblehq/survey/sdk/network/MoshiBuilderProvider.kt

```diff
@@ -3,28 +3,43 @@ package co.nimblehq.survey.sdk.network
 import co.nimblehq.survey.sdk.entity.AnswerEntity
 import co.nimblehq.survey.sdk.entity.QuestionEntity
 import co.nimblehq.survey.sdk.entity.SurveyEntity
-import com.squareup.moshi.JsonAdapter
 import com.squareup.moshi.Moshi
 import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
 import moe.banana.jsonapi2.JsonApiConverterFactory
+import moe.banana.jsonapi2.Resource
 import moe.banana.jsonapi2.ResourceAdapterFactory
 import retrofit2.converter.moshi.MoshiConverterFactory
 
 object MoshiBuilderProvider {
 
-    fun provideJsonApiFactory(): JsonAdapter.Factory = ResourceAdapterFactory.builder()
-        .add(SurveyEntity::class.java)
-        .add(QuestionEntity::class.java)
-        .add(AnswerEntity::class.java)
+    private val listJsonApiClass = mutableListOf<Class<out Resource>>(
```

Comment:
explicit type arguments is redundant
![image](https://user-images.githubusercontent.com/16315358/134778639-2eb53edf-a68b-49d3-981b-c362b8f1a9b2.png)

Reply:
@luongvo I just want to make the list more explicit. But anyway, let remove it when no one need.